### ISSUE TITLE
Shellcheck issues

### ISSUE THREAD

**Issue by jhmartin** [jakewmeyer/Geo#5]
```
In ./geo line 47:
  elif [ "$(expr substr $(uname -s) 1 5)" == "Linux" ]; then
            ^-- SC2003: expr is antiquated. Consider rewriting this using $((..)), ${} or [[ ]].
                        ^-- SC2046: Quote this to prevent word splitting.
                                          ^-- SC2039: In POSIX sh, == in place of = is undefined.
```

**Comment by jakewmeyer** [comment]
So 90% of this looks to have been fixed by the PR's correct?

**Comment by jhmartin** [comment]
@jakewmeyer I split this issue into the ones that I fixed via the PR (now merged) and the ones that weren't. This one wasn't fixed in that PR.

**Comment by jakewmeyer** [comment]
Should be fixed in v0.1.6
